Makefile

```diff
@@ -10,10 +10,7 @@ MANINSTALLDIR=${PREFIX}/share/man
 GO_MD2MAN ?= /usr/bin/go-md2man
 
 ifeq ($(DEBUG), 1)
-  GOGCFLAGS += "-N -l"
-else
-  GOGCFLAGS ?= ""
-  DEBUG ?= 0
+  override GOGCFLAGS += -N -l
```

Comment:
See the comments in https://github.com/projectatomic/skopeo/pull/169/commits/694af0507d9a2471e7a2637d18df53dc1ad35d56 , sorry for not pointing to them.